projects/online_sequence_classification/clustering/clustering.py

```diff
@@ -1,24 +1,75 @@
-class Cluster(object):
-  def __init__(self, id, center):
-    self.id = id
-    self.center = center
-    self.points = []
-    self.size = 0
+# ----------------------------------------------------------------------
+#  Copyright (C) 2016, Numenta Inc. All rights reserved.
+#
+#  The information and source code contained herein is the
+#  exclusive property of Numenta Inc. No part of this software
+#  may be used, reproduced, stored or distributed in any form,
+#  without explicit written authorization from Numenta Inc.
+# ----------------------------------------------------------------------
```

Comment:
This header comes from @scottpurdy 's example: https://github.com/numenta/nupic.research/blob/6c2d2f2265ac712e82e609f02eefd020d9096c53/htmresearch/algorithms/TM.py#L2-L9

@scottpurdy Is that fine or should I replace it with the standard AGPL header? Like https://github.com/numenta/nupic.research/blob/master/projects/capybara/classification/run_htm_network.py#L2-L21